src/index.js

```diff
@@ -4,6 +4,7 @@ import 'bootstrap/dist/css/bootstrap.min.css'
 import './index.css';
 import App from './App';
 import reportWebVitals from './reportWebVitals';
+import Enzyme from 'enzyme'
```

Comment:
I don't think this is needed in this file.

Reply:
Removed line 7 "import Enzyme from 'enzyme'"